app/Http/Controllers/PageController.php

```diff
@@ -454,6 +454,40 @@ public function restoreRevision($bookSlug, $pageSlug, $revisionId)
         return redirect($page->getUrl());
     }
 
+
+    /**
+     * Deletes a revision using the id of the specified revision.
+     * @param string $bookSlug
+     * @param string $pageSlug
+     * @param int $revisionId
+     * @throws NotFoundException
+     * @throws BadRequestException
+     * @return \Illuminate\Http\RedirectResponse|\Illuminate\Routing\Redirector
+     */
+    public function destroyRevision($bookSlug, $pageSlug, $revId)
+    {
+        $page = $this->entityRepo->getBySlug('page', $pageSlug, $bookSlug);
+        $this->checkOwnablePermission('page-update', $page);
```

Comment:
Think it may be better to check `page-delete` permission here instead since it's effectively a related delete operation?